ipaserver/plugins/certmap.py

```diff
@@ -432,18 +434,23 @@ def list_users_by_cert(self, cert):
         Call Users.ListByCertificate interface and return a dict
         with key = domain, value = list of uids
         corresponding to the users matching the provided cert
-        :param cert: DER cert
+
+        :param cert: DER cert, IPACertificate, Certificate
         :raise RemoteRetrieveError: if DBus error occurs
         """
-        try:
+        if isinstance(cert, (x509.IPACertificate, crypto_x509.Certificate)):
```

Comment:
You can only leave `crypto_x509.Certificate` here as `IPACertificate` objects are also its instance.

Reply:
aah, I missed ``@crypto_utils.register_interface(crypto_x509.Certificate)``